### DISCUSSION TITLE
compressai._CXX

### DISCUSSION THREAD

**Discussion by hjl2018** [InterDigitalInc/CompressAI#132] [General] [answered]
I try to run the code, but I find that there is no _CXX in the compressai folder. Can anyone help me? Why?

**Comment by AlbertoPresta** [comment, marked as answer]
I think that you should check in the [cpp_exts](https://github.com/InterDigitalInc/CompressAI/tree/master/compressai/cpp_exts) folder

**Reply by fracape** [reply]
Yes!
And if you faced an error when installing, that has to do with your c++ config, please check this other thread on that:
https://github.com/InterDigitalInc/CompressAI/issues/26
